cost-analyzer/charts/grafana/values.yaml

```diff
@@ -1,3 +1,7 @@
+global:
+  prometheus:
+    enabled: false
+
```

Comment:
Can you share more reasons behind this change? It was set at our standard values.yaml file to enable/disable the deployment of Prometheus chart. Please note that this Grafana's subchart is customized to work with kubecost helm chart, so it may not work independently. Therefore, there are some expected Helm lint errors if you only lint the Grafana chart.